content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -12,6 +12,7 @@ The following [Preview](/about/prisma/releases#preview) feature flags are availa
 - [`referentialIntegrity`](/concepts/components/prisma-schema/relations/referential-integrity) since release [3.1.1](https://github.com/prisma/prisma/releases/tag/3.1.1) - [Submit feedback](https://github.com/prisma/prisma/issues/9380)
 - [`fullTextIndex`](/concepts/components/prisma-schema/indexes#full-text-indexes-mysql-and-mongodb) since release [3.6.0](https://github.com/prisma/prisma/releases/tag/3.6.0) - [Submit feedback](https://github.com/prisma/prisma/issues/10539)
 - [`metrics`](/concepts/components/prisma-client/metrics) since release [3.15.0](https://github.com/prisma/prisma/releases/tag/3.15.0) - [Submit feedback](https://github.com/prisma/prisma/issues/13579)
+- [`tracing`](/concepts/components/prisma-client/tracing) since release [4.2.0](https://github.com/prisma/prisma/releases/tag/4.2.0) - Submit feedback LINK TBA
```

Comment:
This issue link needs to be added

Reply:
Made the issue

```suggestion
- [`tracing`](/concepts/components/prisma-client/tracing) since release [4.2.0](https://github.com/prisma/prisma/releases/tag/4.2.0) - [Submit feedback](https://github.com/prisma/prisma/issues/14640)
```

Reply:
I just noticed that this could be slightly problematic if merged before 4.2.0 is out. The page is hidden, but this section is not. Should maybe be split into a later PR.